src/pages/Home/index.js

```diff
@@ -169,10 +169,10 @@ const Home = () => {
         <div className="col-lg-3 pr-lg-2">
           <DataTile
             icon={isLightTheme ? ActiveSmeshersIcon : ActiveSmeshersIconWhite}
-            title="Active Smeshers"
+            title="Activations"
```

Comment:
Activation transactions

Reply:
"Activations" sounds more concise to me, @pigmej is there a reason you don't like this wording?

Reply:
It was more for the consistincy with the next paragraphs where it was used wrongly as `PoST activations`  hence for consistency `Activation Transaction` but I don't insist here.

Reply:
let's call them activations or, alternatively, "miner identities"